frontend/packages/db-structure/package.json

```diff
@@ -33,9 +35,9 @@
     "lint": "pnpm run '/^lint:.*/'",
     "lint:biome": "biome check .",
     "lint:tsc": "tsc --noEmit",
-    "postinstall": "pnpm run '/^postinstall:.*/'",
-    "postinstall:cp-prism": "cp node_modules/@ruby/prism/src/prism.wasm src/parser/schemarb/prism.wasm",
-    "postinstall:gen-tbls-schema": "node scripts/tblsJsonSchemaToZod.mjs",
+    "gen": "pnpm run '/^gen:.*/'",
+    "gen:cp-prism": "cp node_modules/@ruby/prism/src/prism.wasm src/parser/schemarb/prism.wasm",
+    "gen:gen-tbls-schema": "node scripts/tblsJsonSchemaToZod.mjs",
```

Comment:
If postinstall is set up, it will also be executed when downloading via npm.
Since this is a development use, I changed it to the gen command. 

`gen` is automatically executed by turborepo when necessary.